tests/pyrog/migrations/test_0009_move_column_fk_on_input.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+pytestmark = pytest.mark.django_db
```

Comment:
you forgot `@pytest.mark.migration_test`

Reply:
It wasn't used. Plus, `django_test_migrations adds migration_test marker to each test using migrator_factory or migrator fixture.`